packages/@aws-cdk/aws-ec2/lib/cfn-init-elements.ts

```diff
@@ -247,7 +247,7 @@ export class InitCommand extends InitElement {
     return {
       config: {
         [commandKey]: {
-          command: this.command,
+          command: this.command.length === 1 ? this.command[0] : this.command,
```

Comment:
Not too keen on this, because it changes the meaning of `argvCommand(['/bin/freaky>filename'])` into `shellCommand('/bin/freaky>filename')` which means something different.

Granted, unlikely to occur in practice but still relevant to defend against.